src/ray/rpc/worker/task_size.h

```diff
@@ -0,0 +1,28 @@
+#ifndef RAY_RPC_TASK_SIZE_H
+#define RAY_RPC_TASK_SIZE_H
+
+#include "src/ray/protobuf/direct_actor.grpc.pb.h"
+#include "src/ray/protobuf/direct_actor.pb.h"
+
+namespace ray {
+namespace rpc {
+
+/// The maximum number of requests in flight per client.
+const int64_t kMaxBytesInFlight = 16 * 1024 * 1024;
```

Comment:
Should we consider rename this file to something like `direct_actor_common.h`?  As it's only used for direct actor call, and also it's about RPC stuff and the file name doesn't quite describe its content.